cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -1022,4 +1045,50 @@ public JsonObject getTrigger(String id, String supportedPlatform, String balleri
             }
         }
     }
+
+    /**
+     * Handle unauthorized response.
+     *
+     * @param org  org name
+     * @param body response body
+     * @throws IOException            when accessing response body
+     * @throws CentralClientException with unauthorized error message
+     */
+    private void handleUnauthorizedResponse(String org, Optional<ResponseBody> body)
+            throws IOException, CentralClientException {
+        if (body.isPresent()) {
+            Optional<MediaType> contentType = Optional.ofNullable(body.get().contentType());
+            if (contentType.isPresent() && isApplicationJsonContentType(contentType.get().toString())) {
+                Error error = new Gson().fromJson(body.get().string(), Error.class);
+                throw new CentralClientException("unauthorized access token for organization: '" + org +
+                        "'. reason: " + error.getMessage() +
+                        ". check access token set in 'Settings.toml' file.");
```

Comment:
Let's reword this exception message to add the reason at the end.

Reply:
Added